### PR TITLE
add example for placeholder add table

### PR DESCRIPTION
**Pull request by sampila**

### CODE REVIEW

File: presentation/placeholder-add-table/main.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
+// Copyright UniDoc ehf. All rights reserved.
```
Comment (anovik) [inline presentation/placeholder-add-table/main.go]: Do we need to add year in copyright?

File: presentation/placeholder-add-table/main.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	err := license.SetMeteredKey(os.Getenv(`UNIDOC_LICENSE_API_KEY`))
+	if err != nil {
+		panic(err)
+	}
+}
+
+func main() {
+	ppt, err := presentation.Open("placeholder.pptx")
+	if err != nil {
+		panic(err)
```
Comment (anovik) [inline presentation/placeholder-add-table/main.go]: Maybe log some error, not panic?

**Review by anovik — COMMENTED** [review]
Looks good, added a couple of minor comments

File: presentation/placeholder-add-table/main.go (hunk 1 above)
Comment (sampila) [reply presentation/placeholder-add-table/main.go]: Updated at 8379fee68c74ec161e445c4d6fad986a6d8be406

File: presentation/placeholder-add-table/main.go (hunk 2 above)
Comment (sampila) [reply presentation/placeholder-add-table/main.go]: updated at 8379fee68c74ec161e445c4d6fad986a6d8be406

**Review by anovik — APPROVED** [review]

**Review by 3ace — APPROVED** [review]

**Pull request merged**